### DISCUSSION TITLE
PayTM Payment Gateway

### DISCUSSION THREAD

**Discussion by parvej4u2** [inspireui/support#5074] [Working on Features]
I know that Razorpay is integrated in FluxStore and supports Webview any other payment gateway. But PayTM payment gateway is more popular in our India. PayTM payment gateway is more popular in India because its **UPI and Rupay Dabit Card Transaction Commission is 0% lowest**. If you integrate **PayTM All-In-One SDK** into FluxStore App, I think FluxStore will sell more from India.

PayTM Developer Docs: [https://developer.paytm.com/docs/all-in-one-sdk/](https://developer.paytm.com/docs/all-in-one-sdk/)

PayTM Flutter Docts: https://developer.paytm.com/docs/all-in-one-sdk/hybrid-apps/flutter/?ref=allInOneMerchantIntegration

**Comment by tjain90** [comment]
Yes this one will make app in indian store more flexible. 
It would be really great if this can be added.

**Comment by game4fun2** [comment]
Yes PayTM Payment is very useful for new startup companies...

**Comment by GulsanAktar** [comment]
I have integrated PayTM Payment in all my Indian client's app and website. This is the demand of everyone. It would be great to see it on FluxStore.

**Comment by Haniel5** [comment]
We are implementing this feature and will release on the next 2.5.2 version with this plugin https://wordpress.org/plugins/paytm-payments/
